src/hooks/queries/useMe.ts

```diff
@@ -0,0 +1,20 @@
+import { UserTypeCodec } from 'types/user'
+import { Either } from 'fp-ts/Either'
+import type { DefaultErrorProps } from '@utils/errors'
+import { fetcher } from '@utils/fetcher'
+import * as t from 'io-ts'
+
+const UseMeResponseCodec = t.type({
+  user: UserTypeCodec
+})
+
+type UseMeResponse = t.TypeOf<typeof UseMeResponseCodec>
+
+export type UseMeOutput = t.OutputOf<typeof UseMeResponseCodec>
+
+export const useMe = async (): Promise<Either<DefaultErrorProps, UseMeOutput>> => {
```

Comment:
Não acha que isso aqui devia ser um hook ao invés de só uma função?

Reply:
I fix this in this [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/35/commits/b3f897da8729940a69158281e93fdf538178df36)